NetworkingLab/app/src/main/java/com/justinwells/networkinglab/MainActivity.java

```diff
@@ -0,0 +1,137 @@
+package com.justinwells.networkinglab;
+
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+
+
+
+    public static final String CEREAL_URL
+            = "http://api.walmartlabs.com/v1/search?query=cereal&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny\n";
+    public static final String CHOCOLATE_URL
+            = "http://api.walmartlabs.com/v1/search?query=chocolate&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny";
+    public static final String TEA_URL
+            = "http://api.walmartlabs.com/v1/search?query=tea&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny\n";
+
+    RecyclerView recyclerView;
+
+
+
+    Button cerealButton, chocolateButton, teaButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        cerealButton = (Button) findViewById(R.id.button1);
+            cerealButton.setOnClickListener(onClick);
+        chocolateButton = (Button) findViewById(R.id.button2);
+            chocolateButton.setOnClickListener(onClick);
+        teaButton = (Button) findViewById(R.id.button3);
+            teaButton.setOnClickListener(onClick);
```

Comment:
Please use consistent indenting - not sure why the `setOnClickListener()` lines are indented extra